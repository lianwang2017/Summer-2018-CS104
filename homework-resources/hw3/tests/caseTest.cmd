BUAD
CANST
CSCI
CECS
EE
BISC
HRTHS