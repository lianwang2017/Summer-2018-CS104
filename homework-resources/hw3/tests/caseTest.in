Max Nikias, BUAd
Steven B. Sample, buAd
David Pritchard, CANSt
David Pritchard, CsCI
Mark W. Redekopp, CeCs
Mark W. Redekopp, ee
Merrick Bautista, Cecs
Merrick Bautista, BiSC
Aaron Cote, CsCi
Aaron Cote, HrThS
Nisar Ahmed, Ee
