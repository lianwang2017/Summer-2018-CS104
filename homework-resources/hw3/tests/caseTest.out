BUAD 
Max Nikias
Steven B. Sample

CANST 
David Pritchard

CSCI 
Aaron Cote
David Pritchard

CECS 
Mark W. Redekopp
Merrick Bautista

EE 
Mark W. Redekopp
Nisar Ahmed

BISC 
Merrick Bautista

HRTHS 
Aaron Cote

